src/lib/classes/proxy-deployer.spec.ts

```diff
@@ -23,16 +24,10 @@ describe('UniversalProfile', () => {
     );
     await erc725AccountProxy.initialize(await signer.getAddress());
 
-    await erc725AccountProxy.setData(
-      '0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5',
-      '0xbeefbeef',
-      {
-        from: await signer.getAddress(),
-      }
-    );
-    const data = await erc725AccountProxy.getData(
-      '0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5'
-    );
+    await erc725AccountProxy.setData(ERC725YDataKeys.LSP3.LSP3Profile, '0xbeefbeef', {
```

Comment:
What is '0xbeefbeef' corresponding to ? Is it a constant or is it something that is linked to the test itself ? 
Maybe we could add a comment explaining what is refers to or put it in a variable What do you think ?

Reply:
```suggestion
    const expectedLsp3ProfileData = '0xbeefbeef'
    await erc725AccountProxy.setData(ERC725YDataKeys.LSP3.LSP3Profile, expectedLsp3ProfileData, {
```

Reply:
It is basically to say that it can set any random data key/value pair. I will update the tests.